configtx/msp.go

```diff
@@ -139,6 +145,20 @@ func (m *MSP) isCACert(signingCert *x509.Certificate) error {
 	return fmt.Errorf("signing cert is not a root/intermediate cert for this MSP: %s", m.Name)
 }
 
+func (m *MSP) setMSPConfig(configGroup *cb.ConfigGroup) error {
+	mspConfig, err := newMSPConfig(*m)
```

Comment:
why is this needed?

Reply:
To convert the MSP type to the MSPConfig proto type.